infra.bs

```diff
@@ -280,6 +280,61 @@ quotes and monospace font.
 <a>ASCII lowercase</a> of <var>B</var>.
 <!-- TODO: define string equals? -->
 
+<p>To <dfn export>collect a sequence of <a>code points</a></dfn> means that the following algorithm
+must be run:</p>
+<ol>
+<li><p>Let <var>characters</var> be the set of <a>code points</a> that can be collected.
```

Comment:
It seems the indentation is still incorrect here. But I'm happy to reformat it for you.